### PR TITLE
Enhance consensus round change logs

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

Enhance consensus round change logs

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: consensus/istanbul/core/roundchange.go [hunk 1]
```diff
@@ -122,21 +128,28 @@ func (c *core) handleRoundChange(msg *message, src istanbul.Validator) error {
 
 	if num == numStartNewRound && (c.waitingForRoundChange || cv.Round.Cmp(roundView.Round) < 0) {
 		// We've received enough ROUND CHANGE messages, start a new round immediately.
-		logger.Warn("[RC] Received 2f+1 Round Change Messages. Starting new round")
+		logger.Warn("[RC] Prepare messages received before startNewRound", "round", cv.Round.String(),
+			"len(prepares)", c.current.Prepares.Size(), "messages", c.current.Prepares.GetMessages())
+		logger.Warn("[RC] Commit messages received before startNewRound", "round", cv.Round.String(),
+			"len(commits)", c.current.Commits.Size(), "messages", c.current.Commits.GetMessages())
+		logger.Warn("[RC] Received 2f+1 Round Change Messages. Starting new round",
+			"currentRound", cv.Round.String(), "newRound", roundView.Round.String())
```
Comment (ehnuje) [inline consensus/istanbul/core/roundchange.go:136]: How about merging logs into one?

**Review by ehnuje — COMMENTED** [review]
All the updated logs are handled by `Warn`, however, isn't `Info` more appropriate for some cases?

File: consensus/istanbul/core/roundchange.go (hunk 1 above)
Comment (aidan-kwon) [reply consensus/istanbul/core/roundchange.go:136]: `messages` could contain tens of node addresses. Isn't it too long if they are combined?

File: consensus/istanbul/core/core.go [hunk 2]
```diff
@@ -304,7 +304,7 @@ func (c *core) catchUpRound(view *istanbul.View) {
 	c.roundChangeSet.Clear(view.Round)
 
 	c.newRoundChangeTimer()
-	logger.Trace("Catch up round", "new_round", view.Round, "new_seq", view.Sequence, "new_proposer", c.valSet)
```
Comment (jeongkyun-oh) [inline consensus/istanbul/core/core.go:307]: This is not about log level, but it seems that the `new_proposer` and `c.valSet` are not matched.

**Comment by aidan-kwon** [comment]
> All the updated logs are handled by `Warn`, however, isn't `Info` more appropriate for some cases?

@ehnuje  I agreed that `Info` is more appropriate for some logs. However, I raised the level of the logs which may help to analyze round change process since Cypress occasionally experienced unexpected round changes. All logs modified in this PR are printed when a round change has occurred.

File: consensus/istanbul/core/core.go (hunk 2 above)
Comment (aidan-kwon) [reply consensus/istanbul/core/core.go:307]: @jeongkyun-oh Let's handle the issue in another PR or other channel.

**Review by jeongkyun-oh — APPROVED** [review]

File: consensus/istanbul/core/core.go (hunk 2 above)
Comment (jeongkyun-oh) [reply consensus/istanbul/core/core.go:307]: 👌

**Comment by ehnuje** [comment]
> PR are printed when a round change has occurred.

Understood. LGTM.

File: consensus/istanbul/core/roundchange.go (hunk 1 above)
Comment (ehnuje) [reply consensus/istanbul/core/roundchange.go:136]: If so, it doesn't make much difference 😅 LGTM

**Review by ehnuje — APPROVED** [review]

**Review by KimKyungup — APPROVED** [review]

**Pull request merged**